sdk/forms/README.md

```diff
@@ -10,130 +10,276 @@
->
-> - Payment details are encrypted at rest.
-> - Payment details are encrypted in transit.
-> - Payment details are not stored in logs.
-> - Third party you use to process the payment details is also PCI compliant
-> - Third party libraries are up to date and does not have any known vulnerabilities.
-> - Full audit of your code and processes to get PCI compliance certification.
-> - Full audit log of all access to the payment details.
-> - Full permission and access control on any access to payment details.
-> - Full network and infrastructure isolation of your environment.
->
-> And this is just a partial list...
-
-With Piiano Vault and the Piiano Forms SDK, you can eliminate the need to handle all of this by storing the card details
-in Piiano Vault.
-Vault can store the card details securely and give you a secured tokens you can pass to your backend instead.
-Using the tokens you can then charge the card, without ever exposing the card details to your backend.
-
-Let's take a look at the form:
+Make your forms compliant with PCI or any other privacy regulation by securely managing sensitive data before submitting it to your backend. Using `@piiano/forms`, you can tokenize, encrypt, or store sensitive data directly in Piiano Vault without exposing it to your backend.
 
-```html
-<form action="/cards" method="post">
-  <h1>Add payment details</h1>
-  <label for="card_number">Card Number</label>
-  <input type="text" id="card_number" name="card_number" />
-  <label for="card_holder">Card Holder</label>
-  <input type="text" id="card_holder" name="card_holder" />
-  <label for="card_expiry">Card Expiry</label>
-  <input type="text" id="card_expiry" name="card_expiry" />
-  <label for="card_cvv">Card CVV</label>
-  <input type="text" id="card_cvv" name="card_cvv" />
-  <button type="submit">Add card</button>
-</form>
+## Table of Contents
+
+- [When to Use Piiano Forms](#when-to-use-piiano-forms)
+  - [Example Use Cases](#example-use-cases)
+- [How It Works](#how-it-works)
+- [Setup](#setup)
+- [API](#api)
+  - [createProtectedForm](#createprotectedform)
+    - [Overview](#overview)
+    - [Goal](#goal)
+    - [Usage](#usage)
+  - [createProtectedView](#createprotectedview)
+    - [Overview](#overview-1)
+    - [Goal](#goal-1)
+    - [Usage](#usage-1)
+  - [controlForm](#controlform)
+    - [Overview](#overview-2)
+    - [Goal](#goal-2)
+    - [Usage](#usage-2)
+- [Recommended Vault Authorization Techniques](#recommended-vault-authorization-techniques)
+- [Benefits of Using Piiano Vault with Piiano Forms](#benefits-of-using-piiano-vault-with-piiano-forms)
+- [Getting Started](#getting-started)
+
+## When to Use Piiano Forms
+
+Piiano Forms is ideal for any use case where you want to collect sensitive information (e.g., credit card details, personal contact information) securely and make sure that it does not reach your backend in raw form.
```

Comment:
@ori-shalom - wouldn't you want to include the protected view use cases here as well. This is the overview so it makes sense to me to have the view options also here